src/components/TaskBody.vue

```diff
@@ -578,6 +582,23 @@ export default {
 			this.navigate($event, tab)
 			// In case it is already open, we also have to emit an event to show the tab
 			emit('tasks:open-appsidebar-tab', { tab })
+			this.clicks++
```

Comment:
I think it's fine to simply bind a `@dlbclick` handler here
https://github.com/nextcloud/tasks/blob/868cd6d072c3f68ee4f8008bd06f381457ca5430/src/components/TaskBody.vue#L83
as well, slightly change the signature of the function called to `openAppSidebarTab($event, tab, edit)` and `emit('tasks:edit-appsidebar-notes', $event)` if `edit` is true. That saves us the custom double click implementation. If you double click then, the sidebar tab is technically opened three times, but that doesn't hurt (it's opened twice with the current code already).

Reply:
@blairun I added a commit to your branch. Please check whether that works for you. Then we can merge the PR.

Reply:
Those changes work well for me.  Thanks!